src/lib.rs

```diff
@@ -51,11 +51,11 @@
 //!     let mut example = Example::new();
 //!
 //!     // Assert that everything is inizialized to 0.
-//!     assert_eq!(example.get_a(), false);
-//!     assert_eq!(example.get_b(), 0);
-//!     assert_eq!(example.get_c(), 0);
-//!     assert_eq!(example.get_d(), DeliveryMode::Init);
-//!     assert_eq!(example.get_e(), 0);
+//!     debug_assert_eq!(example.get_a(), false);
+//!     debug_assert_eq!(example.get_b(), 0);
+//!     debug_assert_eq!(example.get_c(), 0);
+//!     debug_assert_eq!(example.get_d(), DeliveryMode::Init);
+//!     debug_assert_eq!(example.get_e(), 0);
```

Comment:
converting test-code `assert` to `debug_assert` is not useful. pls convert back. `debug_assert` only makes sense in "real" code that might even be affected by performance degradation due to the assert in `--release` builds.